docs/deployment/kubernetes.md

```diff
@@ -364,6 +364,11 @@ The following table lists the configurable parameters of the chart and their def
 | `ingress.api.hosts[].host`                         |                                                                | `chart-example.local`          |
 | `ingress.api.hosts[].paths`                        |                                                                | `[]`                           |
 | `ingress.api.tls`                                  |                                                                | `[]`                           |
+| `api.statsd.enabled`                               | Enable statsd metric reporting from gunicorn.                  | `false`                        |
+| `api.statsd.host`                                  | Host URL to receive statsd metrics                             | `null`                         |
+| `api.statsd.hostFromNodeIp`                        | Set as true to get the host information from the node IP       | `false`                        |
```

Comment:
I was going to push some commits somewhere, but I think the only thing I would change is to make this description be:

> Set as true to use the node IP as the statsd host instead